api.ts

```diff
@@ -1035,11 +1035,21 @@ export interface SyncFormulaResult<K extends string, L extends string, SchemaT e
 
   /**
    * Return the list of deleted item ids for incremental sync deletion.
-   * 
+   *
    * TODO(ebo): Unhide this
    * @hidden
    */
   deletedItemIds?: string[];
+
+  /**
+   * Returned by an incremental sync if a full sync is needed to recalculate the items.
```

Comment:
So also this will be ignored if returned by a full sync, right?

Reply:
Yes